packages/babel-compiler/babel-compiler.js

```diff
@@ -91,9 +91,7 @@ function compileWithSwc(source, swcOptions = {}, { inputFilePath, features, arch
         generateLetDeclarations: true,
       }),
     });
-    if (!result.identical) {
-      content = result.code;
-    }
+    content = result.code;
```

Comment:
I was comparing result.identical to the pattern used in other parts with Babel + Reify. However, with SWC + Reify, we should always favor Reify for better compatibility. It seems that "browser-policy-content" triggered an "identical" case, causing the SWC version with top-level await to crash. Reify ensures backward compatibility here.